indexes/zk_ee/ethereum-general-state-tests.yaml

```diff
@@ -7458,9 +7462,14 @@ entries:
       Opcodes_TransactionInit.json:
         hash: '0x26279de3db46d16442d2b49d816c1935'
         enabled: true
+        # TODO: check we won't implement CALLCODE
+        skipCalldatas: ["0x60006000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec66064f25060006000f3"]
+        comment: "CALLCODE is not supported"
```

Comment:
I believe @shamatar wanted to implement it (or even did it already). But we can enable this test later

Reply:
Good catch, it was an old commit, re-enabled them